Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -461,22 +452,54 @@ void ABenchmarkGymGameModeBase::TickUXMetricCheck(float DeltaSeconds)
 
 void ABenchmarkGymGameModeBase::TickActorCountCheck(float DeltaSeconds)
 {
-	const UNFRConstants* Constants = UNFRConstants::Get(GetWorld());
-	check(Constants);
-	FMetricTimer TempActorCheckDelay = Constants->ActorCheckDelay;
 	// This test respects the initial delay timer in both native and GDK
-	if (TempActorCheckDelay.HasTimerGoneOff() &&
-		!TestLifetimeTimer.HasTimerGoneOff())
+	if (TickActorCountTimer.HasTimerGoneOff())
 	{
 		TryInitialiseExpectedActorCounts();
-		
+
 		const UWorld* World = GetWorld();
 		const FString WorkerID = GetGameInstance()->GetSpatialWorkerId();
-		const int32 ActualCountSimulate = GetActorClassCount(ExpectedSimPlayersCount.ActorClass);
-		const int32 ActualCountNPCs = GetActorClassCount(ExpectedNPCsCount.ActorClass);
-		GenerateTotalNumsForActors(WorkerID, World, ExpectedSimPlayersCount, MapAuthoritativeSimPlayers, SmoothedTotalAuthSimPlayers, ActualCountSimulate, false);
-		ReportAuthoritativeNPCs(WorkerID, World, ActualCountNPCs);
-		TempActorCheckDelay.SetTimer(1);
+		if (WorkerID.IsEmpty())
+		{
+			return;
+		}
+		TMap<TSubclassOf<AActor>, FActorCountInfo>& ThisWorkerActorCounts = WorkerActorCounts.FindOrAdd(WorkerID);
```

Comment:
`TMap<TSubclassOf<AActor>, FActorCountInfo>` is used a bit - worth creating a `using` statement for the type